contracts/staking/Staking.sol

```diff
@@ -1323,6 +1323,34 @@ contract Staking is StakingV2Storage, GraphUpgradeable, IStaking {
         return shares;
     }
 
+    /**
+     * @dev Get the epoch when tokens will be unlocked based on a new undelegation.
+     * @param _del Delegation
+     * @param _tokens Amount of tokens to undelegate
+     * @return Epoch when the undelegated tokens unlock
+     */
+    function _getDelegatedTokensLockedUntil(Delegation memory _del, uint256 _tokens)
+        internal
+        view
+        returns (uint256)
+    {
+        uint256 lockingPeriod = delegationUnbondingPeriod;
+        uint256 currentEpoch = epochManager().currentEpoch();
+        if (_del.tokensLocked > 0) {
+            // Calculate weighted average multiplying epochs by 10 to add extra precision
+            // and be able to round the final number up
+            lockingPeriod = MathUtils.weightedAverage(
+                MathUtils.diffOrZero(_del.tokensLockedUntil, currentEpoch).mul(10), // Remaining thawing period
```

Comment:
One thing I am thinking is to make the value 10 as a variable, like `uint256 extraPrecision = 10` 

It is something an auditor mentioned once before, and I liked it, it makes the code more readable. 

I feel it is not worth it to make it a global variable. It is not important enough. Maybe just a local function variable. Wdyt?

Reply:
I'll do it